diff --git a/bin/common b/bin/common
index 7f3cc50..8314d06 100644
--- a/bin/common
+++ b/bin/common
@@ -415,5 +415,5 @@ EOF
         cat <<EOF
 --${boundary}--
 EOF
-    ) | /usr/sbin/sendmail -i -- "$@" || :
+    ) | sendmail -i -- "$@" || :
 }
diff --git a/bin/ftpsync b/bin/ftpsync
index f6881fa..348906c 100755
--- a/bin/ftpsync
+++ b/bin/ftpsync
@@ -33,7 +33,7 @@ set -o pipefail
 # along with this program; if not, write to the Free Software
 # Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
 
-BINDIR=$(dirname $(readlink -f "$0")); . "${BINDIR}/include-git" ## INCLUDE COMMON
+BINDIR=$(dirname $(readlink -f "$0")); ## INCLUDE COMMON
 
 ########################################################################
 ########################################################################
diff --git a/bin/include-install b/bin/include-install
index 79a7dc0..178c152 100644
--- a/bin/include-install
+++ b/bin/include-install
@@ -1,6 +1,6 @@
 # -*- mode:sh -*-
 # vim:syn=sh
 
-CONFDIRS=(~/.config/ftpsync /etc/ftpsync)
-LOCKDIR=~/.local/lock/ftpsync
-LOGDIR=~/.local/log/ftpsync
+CONFDIRS=(@ftpsync_conf@ ~/.config/ftpsync /etc/ftpsync)
+LOCKDIR="${LOCKDIR:-@ftpsync_dir@/locks}"
+LOGDIR="${LOGDIR:-@ftpsync_dir@/log}"
